packages/edit-site/src/components/global-styles/font-library-modal/collection-font-variant.js

```diff
@@ -39,15 +39,15 @@ function CollectionFontVariant( {
 			className="font-library-modal__library-font-variant"
 			htmlFor={ checkboxId }
 		>
-			<Flex justify="space-between" align="center" gap="1rem">
-				<FontFaceDemo fontFace={ face } text={ displayName } />
+			<Flex justify="flex-start" align="center" gap="1rem">
 				<CheckboxControl
 					checked={ selected }
 					onChange={ handleToggleActivation }
 					__nextHasNoMarginBottom={ true }
 					id={ checkboxId }
 					label={ false }
 				/>
+				<FontFaceDemo fontFace={ face } text={ displayName } />
 			</Flex>
 		</label>
```

Comment:
Unrelated to this PR: it's not okay to associate labels and form controls with both a `for` attribute and by wrapping the form control within the label element. I do realize the intent is to make the whole row clickable but this is not okay.
I'm surprised the component allows to set the label prop to `false`.
Also, we do have reusable components for good reasons. One of them is to make sure they're properly labeled. Hardcoding a `<label>` element around the checkbox component that is supposed to internally handle its label defeats the purpose of having reusable components.